source/_components/sensor.airvisual.markdown

```diff
@@ -0,0 +1,112 @@
+---
+layout: page
+title: "AirVisual"
+description: "Instructions on how to use AirVisual data within Home Assistant"
+date: 2017-09-06 12:15
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: airvisual.jpg
+ha_category: Health
+ha_release: 0.53
+ha_iot_class: "Cloud Polling"
+---
+
+The `airvisual` sensor platform queries the [AirVisual](https://airvisual.com/) API for air quality
+data on the nearest city to a latitude and longitude. The resulting information
+creates sensors for the Air Quality Index (AQI), the human-friendly air quality
+level, and the main pollutant of that area. Sensors that confirm to either/both
```

Comment:
**Spelling & Grammar**:
- `confirm` -> `conform`

Reply:
Fixed – thank you!